lsp-mode.el

```diff
@@ -744,7 +744,8 @@ are determined by the index of the element."
                          (const :tag "Struct" Struct)
                          (const :tag "Event" Event)
                          (const :tag "Operator" Operator)
-                         (const :tag "Type Parameter" TypeParameter)))
+                         (const :tag "Type Parameter" TypeParameter)
+                         (const :tag "Other" nil)))
```

Comment:
I was considering changing the 0th entry of `lsp/symbol-kind-lookup` to `Other` or something similar and updating this value to match.